Models/api.model.js

```diff
@@ -2,6 +2,18 @@ const db = require("../db/connection")
 
 exports.selectTopics = () => {
     return db
-    .query("SELECT slug, description FROM topics;")
+    .query("SELECT * FROM topics;")
```

Comment:
Another change here to previous tickets. You should be doing that in a separate pull request as it is outside the scope of this one.